index.js

```diff
@@ -13,13 +17,38 @@ let orderArray = []
 document.addEventListener("click", (e) => {
     // if user clicks the add/remove button, passes the corresponding dataset to
     // the function that is called for the add/remove
+    // if user clicks complete order, modal is revealed
     if(e.target.dataset.add) {
         addItem(e.target.dataset.add)
     } else if(e.target.dataset.remove) {
         removeItem(e.target.dataset.remove)
-    } 
+    } else if(e.target.dataset.complete) {
+        modal.classList.remove("hide")
+    }
+})
+
+// once user confirms payment and orders, preventDefault() is used to
+// stop the page from reloading. Then 2 functions hideElements() and renderMessage()
+// are called. The tasks within each function could all be called within the event
+// listener, as the scope is small, but for best practices created their own functions.
+paymentForm.addEventListener("submit", (e) => {
+    e.preventDefault()
+    hideElements()
+    renderMessage()
 })
 
+// hides the modal and orderSummary sections after payment is taken
+function hideElements() {
+    modal.classList.add("hide")
+    orderSummary.classList.add("hide")
+}
+
+// displays the message with the name the user entered with template literals
```

Comment:
When you make comments, you don't need to go in detail about EVERY detail your function involves (like which functions you used); just keep it general. For instance, for this, I would say: "displays user-specific 'order completion' message"

I'm not going to reject the pull request but just something to keep in mind.